app/assets/javascripts/3dbio_viewer/src/domain/usecases/UploadAtomicStructureUseCase.ts

```diff
@@ -0,0 +1,11 @@
+import { UseCase } from "../../compositionRoot";
+import { EbiDbModelRepository } from "../../data/repositories/EbiDbModelRepository";
+import { UploadOptions } from "../repositories/DbModelRepository";
+
+export class UploadAtomicStructureUseCase implements UseCase {
+    constructor(private dbModelRepository: EbiDbModelRepository) {}
```

Comment:
A use case uses the generic interface, only the composition root knows about the actual implementation, so `: DbModelRepository`.